File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int maxLength = 0;
        Set<Integer> numSet = new HashSet<>();

        for (int num : nums) {
            numSet.add(num);
        }

        for (int num : numSet) {
            if (!numSet.contains(num-1)) {
                int length = 1;
                while (numSet.contains(num + length)) {
                    length++;
                }
                maxLength = Math.max(maxLength, length);
            }
        }
        return maxLength;


    }
}

File: Data Structures & Algorithms/longest-common-prefix/submission-0.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        if(strs is None):
            return ""
        elif(len(strs)==1):
           return strs[0]
        max_e = max(strs, key=len)
        max_length=len(max_e)

        for i in strs:
            if (max_e in i[0:max_length]):
                pass
            else:
                while(max_e not in i[0:max_length]):
                    max_length=max_length-1
                    max_e=max_e[0:max_length]

        if(max_e is None):
            return ""  
        return max_e    
        